.swiftlint.yml

```diff
@@ -34,8 +34,8 @@ identifier_name:
         - no
         - ok
 line_length:
-    warning: 180
-    error: 200
+    warning: 250
+    error: 300
```

Comment:
Keeping the line length to a relatively low number can increase code readability. How many such occurrences we have, and are these occurrences in both code and tests?